ui/apps/platform/src/Containers/AccessControl/AccessScopes/RequirementRowAddKey.tsx

```diff
@@ -98,7 +98,7 @@ function RequirementRowAddKey({
                         className="pf-m-smallest"
                         onClick={handleRequirementKeyCancel}
                     >
-                        <TimesCircleIcon color="var(--pf-global--color--100)" />
```

Comment:
This is the cancel icon for the label selector widget on the AccessControl page. This changes the color from black/black to the default of dark gray/light gray. The former black icon on dark background was very difficult to see.